skipruntime-ts/examples/groups.ts

```diff
@@ -36,37 +35,28 @@ type ServiceInputs = {
 // Type alias for inputs to the active friends resource
 type ResourceInputs = {
   users: EagerCollection<UserID, User>;
-  actives: EagerCollection<GroupID, UserID>;
+  actives: EagerCollection<GroupID, UserID[]>;
```

Comment:
So each `GroupID` is associated with an array of arrays of `UserID`s, why?

Reply:
It lets the example use OneToOneMapper, cutting down on boilerplate.  Charles was looking for some small example showing the feel of writing reactive code, so I tried to tighten this stuff up as much as possible.

One option that could keep the types as before is to define a `OneToManyMapper` and use that here instead.
I just tried that out and it seems useful in general and a bit nicer here in particular -- wdyt?